.storybook/main.ts

```diff
@@ -44,7 +44,7 @@ const config: StorybookConfig = {
 			to: "/static",
 		},
 	],
-	stories: [relativePath("..", "src", "**", "*.stories.@(js|jsx|ts|tsx|mdx)")],
+	stories: ["../src/**/*.mdx", "../src/**/*.stories.@(js|jsx|mjs|ts|tsx|mdx)"],
```

Comment:
just be aware that this _could_ break cross compat w/ other OSes by not using path.join 😞